Src/VimCore/VimSettings.fs

```diff
@@ -149,6 +149,7 @@ type internal GlobalSettings() =
 
     static let GlobalSettingInfoList = 
         [|
+            (AtomicInsertName, "ains", SettingValue.Toggle false)

```

Comment:
Can we use the pattern `(AtomicInsertName, AtomicInsertName, ...)` here?  In general for non-standard options, like `CaretOpacityName`, I don't provide abbreviations.  The reasoning is to try and keep non-standard options as visible as possible.